emeritus-tech/landing-pages-libraries
# [CPT-484] INSEAD: DTCN - IAD stylesheets and markup

🎟 [Jira Ticket CPT-484](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-484)
🎟 [Jira Ticket CPT-486](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-486)

🚢 Deployment Instructions

N/A

✅ QA instructions

** All modals should work as expected (without the modals.js file will only function in fallback mode, meaning that only clicking the X button will close it and ESC key or clicking outside will not)

To test INSEAD **Design thinking and Creativity for Business**

*  Run command npm start
*  Open http://localhost:8080/IDT.html
*  Design should look exactly as original [landing page](https://onlineprogrammes.insead.edu/design-thinking-creativity-business/)

To test INSEAD **Innovation in the Age of Disruption**

*  Run command npm start
*  Open http://localhost:8080/IIAD.html
*  Design should look exactly as original [landing page](https://onlineprogrammes.insead.edu/innovation-in-the-age-of-disruption/)

🖼 Previews

*  **Design thinking and Creativity for Business**

![screencapture-localhost-8080-IDT-html-2020-05-13-14_14_47](https://user-images.githubusercontent.com/58177278/81843601-32b05680-9524-11ea-9580-2190af1fd428.png)

*  **Innovation in the Age of Disruption**

![screencapture-localhost-8080-IIAD-html-2020-05-13-14_14_23](https://user-images.githubusercontent.com/58177278/81843618-3c39be80-9524-11ea-8747-093208fd4ce7.png)
